.github/torchdynamo_commit_hash.txt

```diff
@@ -0,0 +1 @@
+dbb83776c5de185a7a8e2dbafa41d4dc9552d2f9
```

Comment:
nit: It might be better to put these in a subdirectory to avoid cluttering the main `.github` directory if we intend to do more of these

Reply:
Something like `.github/ci_commit_pins` would probably work

Reply:
I think vision is the other one that we should treat this way